lib/movie/screens/movie_details_screen/cubit/movie_details_cubit.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:get_it/get_it.dart';
+import 'package:movie/movie/domain/movie.dart';
+import 'package:movie/movie/domain/movie_service.dart';
+
+part 'movie_details_state.dart';
+
+class MovieDetailsCubit extends Cubit<MovieDetailsState> {
+  MovieDetailsCubit(MovieDetailsState initialState) : super(initialState);
```

Comment:
1) pls add `final Movie movie`
2) delete this `MovieDetailsState initialState`
3) in super you should add `MovieDetailsInitial(movie: movie)`